src/DucksNet.WebUI/Pages/Models/UpdateMedicalRecord.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace DucksNet.WebUI.Pages.Models;
+
+public class UpdateMedicalRecord
+{
+    [Required]
+    [StringLength(36, MinimumLength = 36, ErrorMessage = "Guid must be 36 characters long.")]
+    public string? MedicalRecordId { get; set; }
+
+    [StringLength(36, MinimumLength = 36, ErrorMessage = "Id Appointment must be 36 characters long.")]
+    public string? IdAppintment { get; set; }
```

Comment:
What happens if you pass null to the backend here?

Reply:
it checks in the update method if the guid == guid.empty